departments/mobile/README.md

```diff
@@ -0,0 +1,101 @@
+
+### Chat (open.rocket.chat)
+
+It’s not required to be connected to the chat all the time, but it’s expected that you check it at least twice in the morning and twice in the afternoon so you don’t miss important announcements or important questions.
+
+### Mobile Team Sync
+
+1. There will be a bot every Monday, Tuesday, Thursday and Friday that will ask you in the morning what’s your status (working on, blocked issues, problems you are having, etc). It’s expected that you reply to this bot as soon as you start working that day.
+2. We have calls every Wednesday with iOS and Android team. It’s expected that all of you participate of this meeting (it’s only once a week!) and the duration is between 30min and 45min.
+3. It’s expected that you keep your status in GitHub Projects updated. Every cycle has a project and there’s columns that specifics what should be done, what is in progress, what’s waiting for review and what’s done already.
+
+### Rocket.Chat Company Weekly Calls
+
+Every week the whole company joins a call via Google Meets at 1pm Brasilia Time (duration is no longer than 1h). It’s expected that you are present into this meeting and if not, you **must** inform your team leader why.
+
+### Calendar
+
+Mobile Team has a shared calendar where we put our deadlines and important events to the whole mobile team. Make sure you have it subscribed in your calendar application, you can copy [this url](https://calendar.google.com/calendar?cid=cm9ja2V0LmNoYXRfY3BqcGV2a3BvOXR1NmZxbWczdDNvM2tobjRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ).
+
+It's also expected that you reply to the invitations you are invited for with yes, no or maybe.
+
+### Emails
+
+There are three emails that we can use to communicate internally: `mobile@rocket.chat`, `ios@rocket.chat` and `android@rocket.chat`. If you want to communicate the whole team you can reach `team@rocket.chat`.
+
+It’s expected that you check your email at least twice a day (morning and afternoon) for new invitations, annoucements, etc.
+
+### Forum
+
+There's an [internal category in our forums specific to the Mobile Team discussions](https://forums.rocket.chat/c/internal/mobile). It's encouraged that everyone share ideas and problems there in form of discussion. It's the preferable way to communicate long discussions with your team. All discussions on this category are private.
+
+### Office Vibe
+
+You will receive sometimes emails from Office Vibe questionaries. It’s expected that you reply to all of them in order to help Rocket.Chat improve with your feedback. You have ways to give written feedback to the company from there anonymously and this is very encouraged.
```

Comment:
We do not **encourage** anonymous feedback. In fact, we'd rather have honest and transparent feedback. We should strive to have a company where anyone can give respectful feedback... Of course anonymity is allowed, but we believe its usage should be limited.